src/app/scopes.ts

```diff
@@ -246,6 +260,20 @@ export const PermissionScopes: PermissionScope[] = [
     preview: false,
     admin: false,
   },
+  {
+    name: "ProgramControl.Read.All",
+    description: "Read all programs that user can access",
+    longDescription: "Allows the app to read programs and programControls that the signed-in user has access to in the organization",
+    preview: true,
+    admin: true,
+  },
+  {
+    name: "ProgramControl.ReadWrite.All",
+    description: "Manage programs that user can access",
```

Comment:
Missing 'a': "Manage programs that a user can access"

Reply:
Fixed.